main/controllers/authentication.py

```diff
@@ -0,0 +1,33 @@
+from fastapi import APIRouter
+from sqlalchemy.exc import IntegrityError
+
+from main.commons import exceptions
+from main.commons.exceptions import ErrorCode, ErrorMessage, Unauthorized
+from main.engines.users import add_user, get_user_by_email, verify_user
+from main.schemas.authentication import AccessTokenSchema, LoginSchema, SignUpSchema
+from main.utils import auth
+
+router: APIRouter = APIRouter()
+
+
+@router.post("/register", response_model=AccessTokenSchema)
+async def _add_user(user: SignUpSchema):
+    try:
+        user = await add_user(email=user.email, password=user.password)
+        return {"access_token": auth.create_access_token_from_id(user.id)}
```

Comment:
We can `return AccessTokenSchema(access_token=...)` for better tooling supports, same as below.